remote/package.json

```diff
@@ -14,8 +14,8 @@
     "https-proxy-agent": "^2.2.3",
     "iconv-lite-umd": "0.6.10",
     "jschardet": "3.0.0",
-    "minimist": "^1.2.5",
```

Comment:
although I understand why this is moved, please don't include this change, thanks for the PR

Reply:
I've moved it back to how it was originally.